tests/test_service.py

```diff
@@ -0,0 +1,127 @@
+
+import pytest
+import logging
+
+from plumbum import ProcessExecutionError, local
+from plumbum.cmd import docker
+from plumbum.machines.local import LocalCommand
```

Comment:
This import isn't used.

Please apply copier template https://github.com/copier-org/autopretty and add a pre-commit job to CI. It will handle removing these imports and prettifying all stuff for us.

Let's modernize this stuff properly :wink: